qiita_ware/ebi.py

```diff
@@ -411,14 +411,18 @@ def generate_sample_xml(self, samples=None, ignore_columns=None):
         for sample_name in sorted(samples):
             sample_info = dict(self.samples[sample_name])
 
-            if self._ebi_sample_accessions[sample_name] is None:
-                sample = ET.SubElement(sample_set, 'SAMPLE', {
-                    'alias': self._get_sample_alias(sample_name),
-                    'center_name': qiita_config.ebi_center_name}
-                )
+            sample_accession = self._ebi_sample_accessions[sample_name]
+            if self.action in ('ADD', 'VALIDATE'):
+                if sample_accession is not None:
+                    continue
```

Comment:
Should we raise a warning for samples that were already submitted? So that this gets logged somewhere, or would that information not be useful at all? I'm thinking it might be helpful in the context of admin/help tasks.

Reply:
It has never came up and IMOO I don't think it will be useful here. Note that we display this information (samples in multiple preps) in the GUI in the sample/prep tab.